src/components/notifications/AppSelector/index.tsx

```diff
@@ -124,47 +130,59 @@ const AppSelector: React.FC = () => {
           )}
         </div>
         <div className="AppSelector__wrapper">
-          <Label color="main">Subscribed</Label>
+          {!loading && filteredApps?.length > 0 ? <Label color="main">Subscribed</Label> : null}
```

Comment:
This should work too
```suggestion
          {!loading && filteredApps?.length && <Label color="main">Subscribed</Label>}
```